docs/mpcnc/trucks.md

```diff
@@ -0,0 +1,127 @@
+# Assembling The Trucks
+
+Click on the images to embiggen.
+
+## Bearings
+![!MPCNC Truck parts](https://www.v1engineering.com/wp-content/uploads/2020/06/TParts-scaled.jpg){: width="400"}
+
+* You will need 5 sets of 5/16 x 1 1/2" (M8 x 40mm) Bolts and locking nuts.
+* Two 1/2" sockets and ratchets (or 13mm for M8's).
+* 5 Bearings.
+* Printed Truck and Truck Mirrored parts.
+
+___
+
+## Tension Bolts
+![!MPCNC truck tension bolts](https://www.v1engineering.com/wp-content/uploads/2020/06/Tension-scaled.jpg){: width="400"}
+
+* Pay attention to the direction of the bolt heads.
+* Insert a bearing in the truck and fit the bolts.
+* Just seat the locking nut, these will get adjusted later. Leave these loose.
+
+___
+
+## Stationary Bearings
+![!MPCNC stationary bearings](https://www.v1engineering.com/wp-content/uploads/2020/06/AllBearings-scaled.jpg){: width="400"}
+
+* Fit the last 3 bearings and bolt sets.
+* Notice the bolt orientation, nuts towards the rails.
+* Snug these up. About 7in/lbs.
+
+!!! Caution
+    Using ratchets makes overtightening the bolts very easy. There are small 1mm built in spacers in the printed parts, if you crush these you are about 3x too tight. Loose is better than tight, If you work on your own car it is wayyyyyy less that any bolt you have ever tightened.
+___
+
+## Set The Truck Tension
+![!MPCNC set the tension](https://www.v1engineering.com/wp-content/uploads/2020/06/Set-Tension-scaled.jpg){: width="400"}
+
+* Insert a rail into the truck.
+* You are looking for all bearings to make contact and just a hint of tension. If you tip the rail at a 45 degree angle the trucks should slowly move. Too tight and they will stay, too loose and they will drop right off.
```

Comment:
I like this. Should we warn people that if they are using their hulk arms, they might have the wrong sized tubes? Maybe I'm being paranoid.